ui/app/components/identity/edit-form.js

```diff
@@ -73,12 +73,11 @@ export default Component.extend({
   ).drop(),
 
   willDestroy() {
-    this._super(...arguments);
     const model = this.model;
-    if (!model) return;
-    if ((model.get('isDirty') && !model.isDestroyed) || !model.isDestroying) {
+    if (model && model.get('isDirty') && !model.isDestroyed && !model.isDestroying) {
       model.rollbackAttributes();
     }
+    this._super(...arguments);
   },
```

Comment:
There's a handful of instances like this where super was called before the custom teardown was running which was causing issues since in the tests the application was torn down or in the process before `rollbackAttributes` was completed.